TLM/TLM/Manager/Impl/LaneConnection/LaneConnectionSubManager.cs

```diff
@@ -376,14 +375,16 @@ public class LaneConnectionSubManager :
                     return dir.IsFlagSet(NetInfo.Direction.Backward);
                 }
             }
-            canConnect =
-                IsDirectionValid(ref sourceNetLane, sourceLaneInfo, nodeId, true) &&
-                IsDirectionValid(ref targetNetLane, targetLaneInfo, nodeId, false);
+            canConnect = IsDirectionValid(ref sourceNetLane, sourceLaneInfo, nodeId, true);
```

Comment:
ternary expression for the win

Reply:
notice the `&=` operator. won't fit well with ternary.

Reply:
true! I didn't see that at first.
wouldn't it be better to encapsulate the logic in the isdirectionvalid method itself?
since the method with the change now only does a "90% job".